package-lock.json

```diff
@@ -9,7 +9,12 @@
       "dependencies": {
         "core-js": "^3.6.5",
         "vue": "^3.0.0",
+<<<<<<< HEAD
         "vue-router": "^4.0.0-0"
+=======
+        "vue-router": "^4.0.0-0",
+        "vueperslides": "^3.2.0"
+>>>>>>> b08d163 (Adding vueper slides)
```

Comment:
Your package-lock has strange git artifacts (probably from VScode). I suggest resetting your lock to get rid of these, since it would be pretty gross to do it by hand

Reply:
gotcha, again i think it happened when i rebased!